tests/Twig3FunctionalTest.php

```diff
@@ -63,13 +63,13 @@ public function getData()
 
             // parentheses spacing is not appliable to control structures.
             ['{% if (1 + 2) == 3 %}', null],
-            ['{% for i in (some_array) %}', null],
+            ['{% for i in (some_array) %}{{ i }}', null],
```

Comment:
the variable for the value of a for loop is mandatory in Twig. So it should not be reported as unused if the body of the loop is not empty.